app/src/main/java/org/p2p/wallet/home/ui/crypto/MyCryptoFragment.kt

```diff
@@ -158,6 +162,22 @@ class MyCryptoFragment :
             textViewEmpty.isVisible = isEmpty
             recyclerViewCrypto.isVisible = !isEmpty
         }
+        setAppBarScrollingState(!isEmpty)
+    }
+
+    private fun setAppBarScrollingState(isEnabled: Boolean) {
+        with(binding) {
+            collapsingToolbar.updateLayoutParams<AppBarLayout.LayoutParams> {
+                scrollFlags = if (isEnabled) {
+                    SCROLL_FLAG_SCROLL
+                } else {
+                    SCROLL_FLAG_NO_SCROLL
+                }
+            }
+            if (!isEnabled) {
+                appBarLayout.setExpanded(true, false)
```

Comment:
true false😁

Reply:
nothing we can do here =)
it is java method =)))